standards/code-style.adoc

```diff
@@ -765,16 +767,8 @@ Rx chains of operators require line-wrapping. Every operator must go in a new li
 ----
 fun syncLocations(): Observable<Location> {
     return databaseManager.getAllLocations()
-            .concatMap(object : Func1<Location, Observable<out Location>>() {
-                fun call(location: Location): Observable<out Location> {
-                    return retrofitService.getLocation(location.id)
-                }
-            })
-            .retry(object : Func2<Int, Throwable, Boolean>() {
-                fun call(numRetries: Int?, throwable: Throwable): Boolean? {
-                    return throwable is RetrofitError
-                }
-            })
+            .concatMap({ retrofitService.getLocation(it.id) })
```

Comment:
This should probably just be `concatMap { retrofitService.getLocation(it.id) }`. Kotlin supports direct execution of a lambda in cases where you would be passing a method to execute anyways.